src/context/directory/handlers/clientGrants.ts

```diff
@@ -47,13 +47,14 @@ async function dump(context: DirectoryContext): Promise<void> {
 
   if (clientGrants.length === 0) return;
 
-  const allResourceServers = await context.mgmtClient.resourceServers.getAll({
-    paginate: true,
+  // TODO: Bring back paginate: true
+  const { data: { resource_servers: allResourceServers } } = await context.mgmtClient.resourceServers.getAll({
```

Comment:
Yes, this is important to include. Before Node v4, Auth0 Node SDK didn't natively support pagination and I don't believe that v4 added it either but this needs to be confirmed. The `getAll` function is mutated by the Deploy CLI to support the `paginate` property.

Reply:
**Update:** This could be an opportunity to improve the pagination. I think that with the v4 SDK bringing types that monkey-patching the client like this could be problematic but if you're able to get it work then great. Otherwise may need to break out into a separate function or some other abstraction.

Reply:
**Update 2:** To be clear, the existing code should/will work, the only considerations is to get the types to match up. Because the SDK wasn't typed in v3, we had to create our own `BaseAuth0APIClient` type and augment with the `pool` property. The proposed code in this PR should actually be removed and probably instead replace the `BaseAuth0APIClient` with the v4 SDK's native management client type. There may be additional type mismatches but that _should_ get this to a mostly-functioning state.